pkg/accounting/db.go

```diff
@@ -56,4 +58,12 @@ type DB interface {
 	QueryPaymentInfo(ctx context.Context, start time.Time, end time.Time) ([]*CSVRow, error)
 	// DeleteRawBefore deletes all raw tallies prior to some time
 	DeleteRawBefore(ctx context.Context, latestRollup time.Time) error
+	// CreateBucketStorageTally creates a record for BucketStorageTally in the accounting DB table
+	CreateBucketStorageTally(ctx context.Context, tally BucketStorageTally) error
+	// CreateBucketBandwidthRollup creates a record for BucketBandwidthRollup in the accounting DB table
+	CreateBucketBandwidthRollup(ctx context.Context, rollup BucketBandwidthRollup) error
```

Comment:
I don't think we should have a separate `CreateBucketBandwidthRollup` the queries adding rollups almost always should be upserts. See changes in https://github.com/storj/storj/pull/1586/files#diff-4e3c8fc5bb52e5d050d2e5af986f4c9fR36

Reply:
this will be removed once michal branch is merged in

Reply:
once michal's branch is merged, i will use this method instead https://github.com/storj/storj/pull/1586/files#diff-53d71b1fb61033682939adbe999b0cbbR73

Reply:
I think `CreateBucketBandwidthRollup` is not used anymore